### PR TITLE
[VPAT] Adjustments for Gradeable Views

### PR DESCRIPTION
**Pull request by pbailie**
VPAT adjustments in new/edit gradeable views
* `<label>`, `<fieldset>`, and `<legend>` as needed
* Removed an extraneous `<div>` so to not inappropriately break up a `<fieldset>`.
* `aria-hidden="true"` added where WAVE would detect and report on intentionally hidden elements.  (note that WAVE will still show these elements in its report summary, but screen readers should ignore them)

### CODE REVIEW

File: site/app/templates/admin/admin_gradeable/AdminGradeableCreate.twig [hunk 1]
```diff
@@ -77,15 +77,14 @@
...
-            <div class="option-alt"><a target=_blank href="http://submitty.org/instructor/create_edit_gradeable#types-of-gradeables">What is the type of the gradeable <i style="font-style:normal;" class="helpicon"></i></a></div>    
+            {# (VPAT) fieldset/legend required for gradeable type container radio group #}
+            <fieldset><legend><div class="option-title">Gradeable Type</div></legend>
+            <div class="option-alt"><a target=_blank href="http://submitty.org/instructor/create_edit_gradeable#types-of-gradeables">What is the type of the gradeable <i style="font-style:normal;" class="helpicon"></i></a></div>
             <span id="required_type" >(Required)</span>
         {% else %}
             <b style='font-size:18px'>Gradeable type</b>: <span id="gradeable-type-string">{{ type_string }}</span>
             <br />
         {% endif %}
-    </div>
```
Comment (pbailie) [inline site/app/templates/admin/admin_gradeable/AdminGradeableCreate.twig]: These two lines were removed as they were breaking up the `<fieldset>` opened at line 81 (closes at line 127).  This removal also appears to not have changed the UI layout in my testing.

File: site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig [hunk 2]
```diff
@@ -4,89 +4,92 @@
...
-                    {{ gradeable.isStudentView() and not gradeable.isStudentViewAfterGrades() ? 'checked' : '' }}/> Yes
-            <input type="radio" id="no_student_view" name="student_view" value="false"
-                    {{ not gradeable.isStudentView() ? 'checked' : '' }}/> No
-            <input type="radio" id="yes_student_view_after_grades" name="student_view_after_grades" value="true"
-                    {{ gradeable.isStudentView() and gradeable.isStudentViewAfterGrades() ? 'checked' : '' }}/> When grades are released
-
-            <input type="radio" id="yes_student_view" name="student_view" value="true"
-                    {{ gradeable.isStudentView() ? 'checked' : '' }} hidden/>
+            <fieldset>
+                <legend>View the gradeable on the course home page and download submitted files?</legend><br>
```
Comment (jmcknight98) [inline site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig]: The UI appears to be unaffected except space in between this legend and the associated radio buttons is added.  Can be fixed by just removing the now redundant <br>.  (Same thing exists on line 24)

Comment (pbailie) [reply site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig]: Certainly could be.  I had removed some `<br />` tags in other code for that very reason.  I'll have another look.

Comment (pbailie) [reply site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig]: Done

**Pull request merged**